src/commands.cpp

```diff
@@ -4885,12 +4925,13 @@ bool CommandSetPH::procresult(Result r)
     }
 
     Node *n = client->nodebyhandle(h);
-    if (n)
-    {
-        n->setpubliclink(ph, time(nullptr), ets, false);
-        n->changed.publiclink = true;
-        client->notifynode(n);
-    }
+    assert(n->plink);
+    //if (n)
+    //{
+    //    n->setpubliclink(ph, time(nullptr), ets, false);
+    //    n->changed.publiclink = true;
+    //    client->notifynode(n);
+    //}
```

Comment:
```suggestion
```

Reply:
good idea, i'll remove it (and put the checks in ifdef DEBUG, also we can check ph matches)